src/components/BulletedList.js

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { PALETTE } from "../shared";
+
+const El = styled.ul`
+  position: relative;
+  list-style-type: none;
+  color: ${PALETTE.black60};
+  padding-left: 1.375rem;
+
+  li {
```

Comment:
I would make this `& > li`, you always have to be careful making global overrides on HTML elements and `>` is a good way to do that.